website_mrp_bom/views/website_mrp_bom_templates.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="mrp_bom_page" name="MRP BoM Page">
+        <t t-call="website.layout">
+            <div id="wrap">
+                <div class="bom-page">
+                    <div class="oe_structure"/>
+                    <div class="container">
+                        <div class="row">
+                            <t t-call="website_mrp_bom.mrp_bom_detail_header"/>
+                            <t t-call="website_mrp_bom.mrp_bom_detail_table"/>
+                        </div>
+                    </div>
+                    <div class="oe_structure"/>
+                </div>
+            </div>
+        </t>
+    </template>
+
+    <template id="mrp_bom_detail_header" name="MRP BoM Details Header">
+        <div class="bom-header">
+            <h1 class="col-md-12 text-center" id="mrp_bom_name" t-field="bom.product_tmpl_id.name"/>
+            <div class="col-md-4">
+                <t t-if="bom.product_tmpl_id">
+                    <div t-field="bom.product_tmpl_id.image" t-options='{"widget": "image", "class": "center-block mb16", "max_width": 128}'/>
+                </t>
+                Base Quantity:
+                <div t-field="bom.product_qty"/>
+                <div groups="product.group_uom" t-field="bom.product_uom_id.name"/>
+            </div>
+            <div class="col-md-8 mt32">
+                <t t-if="bom.website_description">
+                    <div t-field="bom.website_description"/>
+                </t>
+            </div>
+        </div>
+    </template>
+
+    <template id="mrp_bom_detail_table" name="MRP BoM Details Table">
+        <div class="bom-table">
+            <div class="col-md-12">
+                <table class="table">
+                    <thead>
+                        <tr>
+                            <th>Product</th>
+                            <th/>
+                            <th>Quantity</th>
+                            <t t-if="has_published_bom">
+                                <th>BoM</th>
+                            </t>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <t t-foreach="bom.bom_line_ids" t-as="line">
+                            <tr style="font-weight: bold;">
+                                <td class="prod-img">
+                                    <span t-field="line.sudo().product_id.product_tmpl_id.image" t-options='{"widget": "image", "class": "center-block mb8", "max_width": 64}'/>
```

Comment:
could make this cleaner and shorter  with a t-set  in loop.

Reply:
I would like to do it, but I wasn't able to use t-set and sudo(). Sudo is needed to images being displayed